src/spread/obj-spread-order.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Spread operation follows [[OwnPropertyKeys]] order
+template: default
+esid: pending
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- setup
+var calls = []
+var o = { get z() { calls.push('z') }, get a() { calls.push('a') } };
+
+//- args
+{...o}
+//- body
+assert.sameValue(calls[0], 'a');
```

Comment:
I think not. [CopyDataProperties](http://sebmarkbage.github.io/ecmascript-rest-spread/#AbstractOperations-CopyDataProperties) uses ```from.[[OwnPropertyKeys]]()```. [OwnPropertyKeys](https://tc39.github.io/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots-ownpropertykeys) has a defined order for its returning list.